plugins/Live/lang/en.json

```
@@ -1,69 +1,69 @@
-        "VisitorLogNoDataMessagePurged": "Likely the data has been purged because the regular deletion of old raw data is enabled and the date for this report is more than %s days old. A super user can change this setting by going to Administration => Privacy.",
+        "VisitsLog": "Visit Log",
+        "DisableVisitsLogAndProfile": "Turn off visit log and visitor profiles",
+        "DisableVisitsLogAndProfileDescription": "Also removes features that depend on these, like the e-commerce log, the segmented visit-log, the real-time map or the real-time widget. May be required to comply with local privacy laws\/guidelines.",
+        "QueryMaxExecutionTimeExceeded": "Could not run the query in time.",
+        "QueryMaxExecutionTimeExceededReasonDateRange": "Maybe the selected date range is too wide?",
+        "QueryMaxExecutionTimeExceededReasonSegment": "Maybe the selected segment didn't match any visits?",
+        "QueryMaxExecutionTimeExceededReasonUnknown": "Please try again. Get in touch with your administrator or support if the problem persists.",
+        "VisitorLog": "Visit Log",
+        "VisitorLogDocumentation": "This table shows the latest visits within the selected date range. Hover over the date of a vist to find out when the most recent one occured. %1$s If the date range includes today, you can see your visitors in real-time. %2$s The data displayed here is always live, regardless of whether and how often you make use of the archiving cron-job.",
+        "VisitorLogNoDataMessagePurged": "The data has likely been purged because the regular deletion of old raw data is on, and the date for this report is more than %s days old. A superuser can change this setting by going to Administration → Privacy.",
         "VisitorProfile": "Visitor profile",
         "VisitorsInRealTime": "Visits in Real-time",
-        "VisitorsLastVisit": "This visitor's last visit was %s days ago.",
+        "VisitorsLastVisit": "Visited %s days ago.",
         "VisitsFrom": "%1$s%2$s visits%3$s from",
         "VisitSummary": "Spent a total of %1$s%2$s on the website%3$s, and viewed %4$s pages%5$s in %6$s visits%7$s.",
         "VisitSummaryWithActionDetails": "Spent a total of %1$s%2$s on the website%3$s, and performed %4$s actions%5$s (%6$s) in %7$s visits%8$s.",
-        "RowActionTooltipDefault": "Show Visits Log segmented by this row",
-        "RowActionTooltipWithDimension": "Show Visits Log segmented by this %s",
-        "RowActionTooltipTitle": "Open segmented Visits Log",
-        "SegmentedVisitorLogTitle": "Visits Log showing visits where %1$s is \"%2$s\"",
-        "OnClickPause": "%s is started. Click to pause.",
-        "OnClickStart": "%s is stopped. Click to start.",
+        "RowActionTooltipDefault": "Show the visit log for each segment of this row",
+        "RowActionTooltipWithDimension": "Show the visit log for each segment of this %s",
```

```suggestion
        "RowActionTooltipWithDimension": "Show visitors for each segment of this %s",
```
This is what I think it does. What does it actually do?

---

Let's assume you are looking at the report of browsers. Each row represents a record like `Chrome` or `Firefox`. It also includes an icon to show all visitors that were using this browser. So it basically opens the visit log, but applies a segment like `browserName==Chrome`. This is what the message is trying to say.
This feature is available for most reports and for reports where the row doesn't have a fixed dimension like `browser`, the default message is used, as it might get segmented by more than one dimension.

Btw. the placeholder of this message is replaced with the name of the dimension (e.g. Browser, Country, ...)